src/com/sap/piper/JenkinsUtils.groovy

```diff
@@ -231,7 +231,13 @@ void handleStepResults(String stepName, boolean failOnMissingReports, boolean fa
     } else if (reportsFileExists) {
         def reports = readJSON(file: reportsFileName)
         for (report in reports) {
-            archiveArtifacts artifacts: report['target'], allowEmptyArchive: !report['mandatory']
+            String target = report['target'] as String
```

Comment:
where does target come from? might it start with `./../`? In that case it would be wrong. also I would like to understand why archiveArtifact fails when starting with `./`.

Reply:
It it starts with ./../ it would not work anyway, since archiveArtifact is documented to only archive files in the workspace, with the current dir being the workspace. Can't say why it doesn't accept patterns starting with ./. Haven't looked up the code and the documentation is sparse as usual with Jenkins steps.

Reply:
Actually, replacing `./../` with `../` should not change the meaning. Of course replacing `.//...` with `/...`would change it. But the point is that if any target path was passed to archiveArtifact like this, it was already broken and did not work.